pos_sale_order_delivery_carrier/models/delivery_carrier.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class DeliveryCarrier(models.Model):
+    _inherit = "delivery.carrier"
+
+    available_in_pos = fields.Boolean(
+        string="Available in POS",
+        help="Check if you want this shipping method to appear in the Point of Sale",
+        default=True,
+    )
```

Comment:
Since you have a `delivery_carrier_ids` field on `pos.config`, this setting seems too much IMO, don't you think?

I think it's easier for users if there's only 1 configuration point